samples/refreshAccessToken.js

```diff
@@ -1,112 +0,0 @@
-// Copyright 2017, Google, Inc.
```

Comment:
Why was this removed?

Reply:
The `refreshAccessToken` method was deprecated in 2.0 :) Users literally never need to call it.